mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala

```diff
@@ -22,7 +22,13 @@ trait ArgCompletions { this: MetalsGlobal =>
     val method: Tree = typedTreeAt(funPos)
     val methodSym = method.symbol
     lazy val baseParams: List[Symbol] =
-      if (method.tpe == null) {
+      if (methodSym.isModule) {
+        methodSym
+          .localCompanion(funPos)
```

Comment:
If the case class was defined inside a method, we can't use `methodSym.companionClass`